### PR TITLE
Issure56

### PR DESCRIPTION
**Pull request by xyl96**
Improving documentation format for chemical dissolution

### CODE REVIEW

File: docs/src/denudation/chemical.md [hunk 1]
```diff
@@ -75,16 +85,28 @@ However, to solve this equation we still need to know $c(z)$.
 
 If assuming the initial percolating water has $c(0) = 0$, then we could get the following equation (as $c$ is related to depth):
 
-$$c(z) = c_{eq} * (1 - e^{(-z/\lambda)})$$
+$$c(z) = c_{eq}\ (1 - e^{(\frac{-z}{\lambda})})$$
```
Comment (jhidding) [inline docs/src/denudation/chemical.md:88]: Superfluous parentheses. In superscript like this I prefer a/b notation for fractions

**Review by jhidding — CHANGES_REQUESTED** [review]

**Comment by jhidding** [comment]
Please also run entangled

File: docs/src/denudation/chemical.md [hunk 2]
```diff
@@ -52,7 +52,17 @@ function karst_denudation_parameters(temp::Float64)
         activity_Alk=10^(-A * sqrt(IA) / (1 + 5.4 * 10^(-8) * B * sqrt(IA))))
 end
 ```
+and ```equlibrium``` function to calculate the $[Ca^{2+}]_{eq}$:
```
Comment (EmiliaJarochowska) [inline docs/src/denudation/chemical.md:55]: should be `equilibrium` (typo)

File: docs/src/denudation/chemical.md [hunk 3]
```diff
@@ -6,20 +6,20 @@ Limestone is made of $CaCO_3$, easily dissolved. This depends mainly on precipit
...
+Herein $\frac{dh}{dt}$ is the chemical weathering rate and the unit is in m/s.
 Other parameters are defined as: $q_i$ is the discharge of water at a certain cell. $A_i$ is the surface area of the cell. If we assume there would be no surface water on land, $q_i$ reduces to precipitation – evaporation. Let’s set it to 400 mm/y for now. Therefore equation 1 could be reduced to Equation 2.
 
-$(dh/dt) = 0.001 * κ_c * I$
+$$\frac{dh}{dt} = 0.001\ \kappa_c\  I$$
 
 Where $I$ is runoff (mm/y?). The parameter $\kappa_c$ is dimensionless and should be described by equation 3:
 
-$kc = 40 * 1000 * [Ca^{2+}]_{eq}/ρ$
+$$\kappa_c = 40\times 1000\ \frac{[Ca^{2+}]_{eq}}{ρ}$$
```
[the remark this answers is not in the document]
Comment (EmiliaJarochowska) [reply docs/src/denudation/chemical.md:16]: I think this was on me, why is `\times` wrong?

[the remark this answers is not in the document]
Comment (jhidding) [reply docs/src/denudation/chemical.md:16]: It's not wrong, just not common where I come from

**Pull request merged**